src/rtf-plugins/fixture-managers/yarpserver/yarpserverfixture.cpp

```diff
@@ -49,7 +49,13 @@ class YarpNameServer : public FixtureManager
     {
 
         yServer.configure(argc, argv);
-        return yServer.start();
+        yServer.start();
+        double time = yarp::os::Time::now();
```

Comment:
I suggest to always use the system clock here:
~~~
double time = yarp::os::SystemClock::nowSystem()
~~~